Presentation.Web/Controllers/External/V2/ItSystems/ItSystemSensitivePersonalDataTypeV2Controller.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Core.ApplicationServices.OptionTypes;
+using Core.DomainModel.ItSystem;
+using Core.DomainModel.ItSystemUsage;
+using Presentation;
+using Presentation.Web;
+using Presentation.Web.Controllers;
+using Presentation.Web.Controllers.External;
+using Presentation.Web.Controllers.External.V2;
+using Presentation.Web.Controllers.External.V2.ItSystemUsages;
+using Presentation.Web.Controllers.External.V2.ItSystemUsages.SystemRelations;
+using Presentation.Web.Models.External.V2;
+using Presentation.Web.Models.External.V2.Request;
+using Presentation.Web.Models.External.V2.Response;
+using Swashbuckle.Swagger.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Http;
+
+namespace Presentation.Web.Controllers.External.V2.ItSystemUsages.SystemRelations
+{
+    [RoutePrefix("api/v2/it-system-sensitive-personal-data-types")]
+    public class ItSystemSensitivePersonalDataTypeV2Controller : BaseOptionTypeV2Controller<ItSystem, SensitivePersonalDataType>
```

Comment:
mjaeh det der er jeg nu ikke så begestret for. Det er jo på ItsystemUsage man vil sætte typen så jeg tror vi har fat i noget andet her. Dét der er jo en ITsystem option. På ItsystemUsage ligger der:

![image](https://user-images.githubusercontent.com/13116889/123582082-770cbd80-d7dd-11eb-9a0e-c0fc5843f42d.png)

Det skal lige undersøges nærmere

Reply:
Jeg kan se at det hidtil har været sådan at SensitiveDataLevel har været sat på både det ene og det andet objekt (it system og usage) og derfår henter man altid data igennem AttachedOptionsController.... Ift. API er dette dog uden tvivl. it-system-usage-***

og controlleren er ITSystemUsage****

også selvom det stadig er noget rod inde bagved for det skal API'et ikke være et spejl af

Reply:
ItSystemUsageSensitivePersonalDataTypeV2Controller